ethpm_cli/parser.py

```diff
@@ -512,3 +513,27 @@ def list_action(args: argparse.Namespace) -> None:
 )
 add_ethpm_dir_arg_to_parser(list_parser)
 list_parser.set_defaults(func=list_action)
+
+
+#
+# ethpm examine
+#
+
+
+def examine_action(args: argparse.Namespace) -> None:
+    config = Config(args)
+    examine_manifest(config.manifest_path)
+
+
+examine_parser = ethpm_parser.add_parser(
+    "examine",
+    help="Examine the contents of the manifest located at the provided manifest path.",
+)
+examine_parser.add_argument(
+    "--manifest-path",
```

Comment:
Might be good to use a positional argument here instead of an option so that this command can be called as `ethpm examine ./path/to/manifest.json`